lockc/src/bin/lockcd.rs

```diff
@@ -1,13 +1,31 @@
-use std::path;
+use std::{env, path};
 
 use chrono::prelude::*;
+use log::debug;
+use simplelog::{ColorChoice, ConfigBuilder, LevelFilter, TermLogger, TerminalMode};
 
 fn main() -> anyhow::Result<()> {
-    let sys_lsm_path = path::Path::new("/sys")
-        .join("kernel")
-        .join("security")
-        .join("lsm");
-    lockc::check_bpf_lsm_enabled(sys_lsm_path)?;
+    let log_level = match env::var("LOCKC_DEBUG") {
+        Ok(_) => LevelFilter::Debug,
+        Err(_) => LevelFilter::Info,
+    };
```

Comment:
I assume this is a boolean setter, not a level?

Reply:
I was assuming it's a level. Was my assumption wrong?

Reply:
I ask because I think you always set the enum to either Debug or Info based on whether or not that ENV's value is non-empty or exists. Thus, it seems like it behaves like a bool (true/false for debug mode)

Reply:
Yes, exposing it as a bool env variable was my intention.